### PR TITLE
Add badges with links to relevant pages

### PR DESCRIPTION
**Pull request by sldblog**
## What does this pull request do?

Our new hires in another team in the service area have highlighted that it is hard to find something related to an application unless we know where to look.

One of the ideas to deal with this difficulty is to make the application Readmes the starting point for everything.

In the above spirit, this pull request introduces new badges that will link directly to:

- Where to find the application runbook.
- Where to find the runtime exceptions.
- Where to find the runtime logs.
- Where to find the feature tests.
- Where to find the continuous integration system, with a representation of the latest production build.

Screen Shot 2019-06-28 at 13 59 21

## Any other changes that would benefit highlighting?

Not at this time.

## Checklist

- [ ] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title" **Not applicable.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+[![Runbook](https://img.shields.io/badge/runbook-edit-172B4D.svg?logo=confluence)](https://dsdmoj.atlassian.net/wiki/spaces/laagetaccess/pages/1388150812/Applications+details)
+[![Exceptions](https://img.shields.io/badge/exceptions-view-FB4226.svg?logo=sentry)](https://sentry.service.dsd.io/mojds/prod-public/)
```
Comment (sldblog) [inline README.md]: 🙋‍♂ I'm using the colours corresponding to the icon from https://simpleicons.org/. (The only exception is GitHub, which would be black...)

**Comment by said-moj** [comment]
This is awesome!

**Comment by sldblog** [comment]
📝 Not deploying to staging as it's only readme changes.

**Comment by sldblog** [comment]
🎩 🙇

**Comment by sldblog** [comment]
📝 The badge, unfortunately, does not show if we have pending deployments on `master` workflows.

**Pull request merged**